Source/WebCore/dom/DocumentMarkerController.cpp

```diff
@@ -560,6 +570,11 @@ void DocumentMarkerController::removeMarkers(OptionSet<DocumentMarker::Type> typ
         return;
     ASSERT(!m_markers.isEmpty());
 
+#if ENABLE(UNIFIED_TEXT_REPLACEMENT)
+    if (types.contains(DocumentMarker::Type::UnifiedTextReplacement))
+        m_unifiedTextReplacementAnimationTimer.stop();
+#endif
```

Comment:
This should be moved after the loop and consult `removedMarkerTypes` rather than `types`. The filter function might result in nothing being removed.